src/Scaffold/Console/model/model.stub

```diff
@@ -22,6 +24,39 @@ class {{studly_name}} extends Model
      */
     protected $fillable = [];
 
+    /**
+     * @var array Rules for fields
+     */
+    public $rules = [];
+
+    /**
+     * @var array Cast fields
+     */
+    protected $casts = [];
+
+    /**
+     * @var array Json fields
+     */
+    protected $jsonable = [];
+
+    /**
+     * @var array Append attributes
+     */
+    protected $appends = [];
+
+    /**
+     * @var array Hide attributes
```

Comment:
Comment should be `Attributes to be removed from the API representation of the model (ex. toArray())`